src/components/new-request/search.vue

```diff
@@ -1,14 +1,12 @@
 <template>
-  <v-container fluid id="search-container" class="copy-normal pa-10">
-    <v-row no-gutters>
-      <v-col cols="12" class="pt-0 font-weight-bold h6">
+  <v-container fluid id="search-container" class="copy-normal pt-10 px-10 pb-12">
+    <v-row>
+      <v-col cols="12" class="font-weight-bold h6">
         Get started by selecting an action: [{{ getRequestActionCd }} / {{ getLocation }} / {{ entity_type_cd }}]
       </v-col>
-    </v-row>
 
-    <v-row class="mt-6">
```

Comment:
I got rid of the "second row". Most sections here are simply columns, which automatically wrap or stack depending on screen width. Eg,

![image](https://github.com/bcgov/namerequest/assets/10002889/bf3a709d-a129-48ab-9ec7-40cd9826c0b8)